main.cpp

```diff
@@ -1,19 +1,51 @@
-#include <iostream>
-#include <cstdlib>
-
-int getNumber( int min, int count )
-{
-    return min + rand() % count ;
-}
-
+#include <iostream>  
+#include<Windows.h>
+#include<math.h>
+//using namespace std;
 int main()
 {
-    srand( time( nullptr ) );
+	setlocale(LC_ALL, "");
+	srand(time(0));
+	int faster = 0, fast = 0, sum1 = 0, sum2 = 0, same = 0;
+	do
+	{
+		for (int i = 0;i < 100;i++)
+		{
+			if (i % 2 == 0)
+			{
+				faster = rand() % 6 + 1;
+				fast = rand() % 6 + 1;
+				sum1 += faster + fast;
+				std::cout << faster << "/" << fast << "\n";
+				Sleep(10);
+				system("Cls");
+			}
+			else
+			{
+				faster = rand() % 6 + 1;
+				fast = rand() % 6 + 1;
+				sum2 += faster + fast;
+				std::cout << faster << "/" << fast << "\n";
+				Sleep(10);
+				system("Cls");
+			}
+		}
+		std::cout << "����� ������� ������� ������:" << sum1 << "\n";
+		std::cout << "����� ������� ������� ������:" << sum2 << "\n";
+
+		if (sum1 > sum2)
+		{
+			same = sum1 - sum2;
+			std::cout << "��������� ������� ������� ������ ������ ��" << same << "\n";
+		}
+		else if (sum2 > sum1)
```

Comment:
```suggestion
		if (sum1 > sum2)
		{
			same = sum1 - sum2;
			std::cout << "��������� ������� ������� ������ ������ ��" << same << "\n";
		}
		else if ( sum1 < sum2 )
```
Более читаемый код, если не менять местами переменные.